spec/features/api_spec.rb

```diff
@@ -1,9 +1,9 @@
 require "spec_helper"
 
-RSpec.describe "Suspend a new project with --api flag" do
+RSpec.describe "Suspend a new project with --api flag", type: :feature, autoclean: false do
```

Comment:
The `spec/features` directory had two types of test:

1. `type: :generator`
2. Tests with no type, like this one

However,:

- All tests from this directory are actually feature tests;
- `type: :generator` tests were actually full-blown integration tests and not the more isolated generator tests we're striving for here.

What I'm doing here is tagging every test inside `spec/features` with `type: :feature`.

`autoclean: false` is temporary and will be removed in a future PR. It should be used for tests that use a `before(:all)` hook to setup the project directory, which will be a characteristic of all feature tests in the future.

Feature tests that don't use `before(:all)` (former "generator tests") are still relying on `autoclean` to keep backward compatibility, but they will be converted to more unitary "generator tests" as the work progresses.